src/pages/Skills/index.js

```diff
@@ -35,13 +35,6 @@ const useStyles = makeStyles((theme) => ({
     margin: "auto",
     padding: theme.spacing(3)
   },
-  heading: {
```

Comment:
made sense to me to keep all the heading styling in the typography object in App.js as all the pages should have the same heading (including same padding etc) and this avoids duplication on individual pages, but not sure whether padding is a typography concern? let me know what you think is best

Reply:
yep agree sounds goof

Reply:
does the padding need to be added back in here though?

Reply:
yeah i think we'd have to add padding back here if we took it out of typography. and to all the other pages where we want a similar heading (which is basically all pages).